src/tests/ftest/util/daos_io_conf.py

```diff
@@ -59,23 +58,21 @@ def __init__(self, path="", env=None):
         self.dkeys = FormattedParameter("-d {}")
         self.record_size = FormattedParameter("-s {}")
         self.obj_class = FormattedParameter("-O {}")
-        self.filename = BasicParameter(None, "testfile")
+        self.filename = BasicParameter(None, filename)
 
     def run_conf(self):
         """Run the daos_run_io_conf command as a foreground process.
 
         Raises:
-            CommandFailure: if there is an error running the command
+            None
 
         """
         command = " ".join([os.path.join(self._path, "daos_run_io_conf"),
                             self.filename.value])
-        try:
-            return run_command(
-                command, self.timeout, self.verbose, env=self.env)
-        except DaosTestError as error:
-            raise CommandFailure(error)
 
+        manager = Orterun(command)
```

Comment:
does this really need to be launched with orterun?
does it even use MPI?

Reply:
It calls MPI_Init, MPI_Barrier, etc.

Reply:
Yes it's based on MPI and does use the  MPI_Init, MPI_Barrier, etc..

Reply:
OK thanks for confirming